community/cypher/cypher-planner-3.4/src/test/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/idp/CartesianProductsOrValueJoinsTest.scala

```diff
@@ -126,12 +130,12 @@ class CartesianProductsOrValueJoinsTest
         case RegularPlannerQuery(queryGraph, _, _) if queryGraph.patternNodes == Set("c") => 3000.0
         case _ => 100.0
       }
-    }.withLogicalPlanningContext { (cfg, ctx) =>
-      val kit = ctx.config.toKit(ctx)
+    }.withLogicalPlanningContext { (cfg, ctx, solveds, cardinalities) =>
```

Comment:
This looks like it would work with Fake Attributes.

Reply:
Nope